packages/js/email-editor/package.json

```diff
@@ -0,0 +1,91 @@
+{
+	"name": "@mailpoet/email-editor",
```

Comment:
Is that the correct package name? I'd expect that only `@woocmmerce/*` packages are introduced in the monorepo.

Reply:
Also, I noticed we are missing CI-config: here's [an exmple](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/components/package.json#L177).